packages/onboarding/jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
 	preset: '../../test/packages/jest-preset.js',
 	testEnvironment: 'jsdom',
+	transformIgnorePatterns: [ 'node_modules/(?!gridicons)(?!.*\\.svg)' ],
```

Comment:
Can you explain this a bit?

Reply:
Yes, this resolves the broken unit tests. Because jest doesn't accept SVG it breaks whenever the GridIcons component is used. This was the issue you helped me resolve during our meetup. However, instead of adding the `mock` I noticed that a few other `jest.config` files were using this ignore statement to ignore the SVG being loaded. It is in the packages jest config - https://github.com/Automattic/wp-calypso/blob/7fe7e962ea85fab34bf7abaa846c646e9e004e8a/packages/components/jest.config.js

Reply:
Oooh this is a much cleaner solution. Thanks!